oeps/oep-0002-bp-repo-metadata.rst

```diff
@@ -60,34 +69,86 @@ Each repo will include a file ``openedx.yaml``, with the following keys:
     -  If the ``state`` is ``False`` a ``reason`` value must be included that explains why.
     -  If the ``state`` is ``True`` or ``applicable`` is ``False`` the ``reason`` can optionally provide more information. The ``reason`` value will be displayed in reporting tools. If an OEP isn't listed in the ``oeps`` dictionary, then it is assumed to be ``False``, unless the reporting tool can auto-detect accordance.
 
-``track-pulls``: obsolete
-    This key is obsolete, please remove if found.
-
-``openedx-release``: defined by :doc:`OEP-10 <oep-0010-proc-openedx-releases>`.
+``openedx-release``: defined by :doc:`OEP-10 <oep-0010-proc-openedx-releases>` (optional)
     Define this key if your repo is an application or IDA that is part of Open edX releases.  Omit this key if your repo is a library, or is not part of Open edX releases.  See :doc:`OEP-10 <oep-0010-proc-openedx-releases>` for details.
 
 ``archived``: boolean (optional)
     If ``True``, this specifies that this repository is archived and no longer maintained by edX.
 
+Obsolete Keys
+*************
+
+``track-pulls``:
+    This key is obsolete, please remove if found.
+
+``supporting_teams``: list of strings
+    This is an obsolete key that was used to specify Github handles for teams supporting the owner of the repository. This is now deprecated in favor of the updated ``owner`` dictionary. As a reasonable migration of this key, set ``owner.type`` to *team*, ``owner.value`` to the first value in ``supporting_teams``, and then remove ``supporting_teams``.
+
 Tags
 ----
 
 The ``tags`` structure is very open ended but there are a few tags that are currently in wide use and are provided here so that they can be used consistently.
 
 ``webservice``:
-    Indicate that this repository is the root of a webservice.  eg. It can serve HTTP content.
+    The repository is the root of a webservice that serves HTTP content.
 
-``library``:
-    The repository is an installable package that is reusable but doesn't run standalone. eg. a pip installable pyton package
+    Code runs in production web servers and has publicly accessible views/APIs for end-users.
+
+    eg. credentials
+
+``frontend-app``:
+    The repository primarily contains code for a frontend application.
+
+    Code runs in production devices (such as web browsers and mobile devices) and is publicly accessible by end-users.
+
+    eg. frontend-app-profile, edx-app-ios
+
+``framework``:
+    The repository contains shared functionality that form a common framework used by multiple services or apps in the platform.
```

Comment:
form -> forms (but maybe you really want "forms part of"?)

Reply:
Updated to `forms part of`. Thanks!